backend/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>it.geosolutions.mapstore</groupId>
   <artifactId>mapstore-backend</artifactId>
   <packaging>jar</packaging>
-  <version>1.0-SNAPSHOT</version>
+  <version>1.0-2020.02</version>
```

Comment:
Seems good. 
One question: Is not clear to me why 1.0-2020.02.
What is the naming convension intended of this?
I think one of the following is more rational:
 - 1.0 and for 2020.02 
 - 1.1 or 2020.03
 and so on...

Or:
- 1.0-2020.02
- 1.0-2020.03
and so on.

I'm sure there is a reason behind it but I can't get it.

Reply:
The idea is to make the relationship between the MapStore version and the backend version clear, so I think it is worth having 2020.02 as part of the version.
Removing 1.0 (so having only 2020.02) would work the same, feel free to use the versioning schema you prefer.

Reply:
Yes, I supposed. I said because I've seen your edit on the issue, and it was not clear: 

> **2020.02.02** and **2020.02.xx**
> - [ ] print-lib v2.0-2020.02
> - [x] http-proxy v1.1.0
> - [x] geostore v1.5.0
> - [ ] mapstore-backend v1.0-2020.02
> 
> **2020.03.00** and **2020.03.xx** (as soon as the branch is created, it doesn't exists yet)
> - [ ] print-lib v2.0.0
> - [ ] http-proxy v1.1.0
> - [ ] geostore v1.6.0
> - [ ] mapstore-backend v1.0.0
> 
@tdipisa what do you think? Having this makes sense:
**2020.02.02** and **2020.02.xx**
- [ ] print-lib v2.0-2020.02
- [x] http-proxy v1.1.0
- [x] geostore v1.5.0
- [ ] mapstore-backend **v2020.02**

**2020.03.00** and **2020.03.xx** (as soon as the branch is created, it doesn't exists yet)
- [ ] print-lib v2.0.0
- [ ] http-proxy v1.1.0
- [ ] geostore v1.6.0
- [ ] mapstore-backend **v2020.03**

Reply:
@offtherailz @mbarto in my opinion it would be better to have something as @mbarto suggested so that we are able to better provide minor versions of the backend if needed. Therefore, having something like @mbarto provided in this PR is fine in my opinion.

Reply:
good point.
so I suggest 2020.02-1.0 etc...